HACKING.md

```diff
@@ -24,7 +24,7 @@ To build this project you must first install several command line utilities and
 - [`make`](https://www.gnu.org/software/make/) - Make build system
 - [`mvn`](https://maven.apache.org/index.html) (version 3.5 and above) - Maven CLI
 - [`helm`](https://helm.sh/) - Helm Package Management System for Kubernetes
-    - After installing Helm be sure to run `helm init`.
+    - If you are using the Helm version 2 charts, after installing the Helm CLI, ensure you run `helm init` to configure to your cluster.
```

Comment:
This doc is targeted to developers - so I guess they need to also have both versions of Helm installed as `helm2` and `helm3` to be able to run `make all` etc.? I guess that should be mentioned here.